<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfflineLobbyButton</class>
 <widget class="QWidget" name="OfflineLobbyButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#OfflineLobbyButton {
	background-color: rgba(255, 255, 255, 255);
}

QFrame#offlineButtonFrame {
	background-color: rgba(255, 255, 255, 255);
    border: 1px solid rgb(100, 100, 100); 
    border-radius: 0px;
}

QFrame#offlineButtonFrame:hover {
	background-color: rgb(240, 240, 240);
}

QLabel#nameLabel {
	font-family: &quot;Arial&quot;;
	font-size: 20px;
	color: rgb(115, 115, 115);
	font-weight: bold;
}

QLabel#descriptionLabel {
	font-family: &quot;Arial&quot;;
	font-size: 16px;
	color: rgba(130, 130, 130, 200);
	font-weight: bold;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="offlineButtonFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="descriptionLabel">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
